src/index.ts

```diff
@@ -30,7 +30,7 @@ async function main() {
   }
 
   console.log({ eventName, sha, headSha, branch, owner, repo, GITHUB_RUN_ID });
-  const token = core.getInput('access_token', { required: true });
+  const token = core.getInput('access_token');
```

Comment:
default value will always provide it, but should still be required.

Reply:
No need to have extra checks if you have a default value